app/views/models/index.html.erb

```diff
@@ -16,7 +16,7 @@
     <% @models.each do |model| %>
       <tr>
         <td class="display-two-pane ellipsis">
-          <%= link_to_if_authorized trunc_with_tooltip(model.name),
+          <%= link_to_if_authorized model.name,
```

Comment:
This was actually a regressing introduced in https://github.com/theforeman/foreman/commit/aa50ab15ff091f132be81861f03c6a9919f2410d#diff-d0a1a867ec446b231c4e11653e019a02R19, just happened to notice it when working on this issue.